changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+bump: patch
```

__issue:__ Sorry, I missed this the first time
```suggestion
- bump: patch
```